server/src/test/java/org/apache/druid/client/BrokerInternalQueryConfigTest.java

```diff
@@ -33,18 +33,20 @@
 import org.apache.druid.guice.LazySingleton;
 import org.apache.druid.jackson.DefaultObjectMapper;
 import org.apache.druid.segment.TestHelper;
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.util.HashMap;
 import java.util.Map;
 
-public class BrokerInternalQueryConfigTest
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class BrokerInternalQueryConfigTest
```

Comment:
Does JUnit5 require/recommend the test classes and methods to be package-protected? If not, we can continue to keep the classes and methods as `public` to reduce the footprint of this PR.